eng/AzSdk.test.reference.targets

```diff
@@ -12,7 +12,4 @@
     <!-- This is needed for discovering tests in test explorer -->
     <PackageReference Include="System.Runtime.InteropServices" />
   </ItemGroup>
-  <ItemGroup> 
```

Comment:
I don't think these are doing anything for our projects I think it is only for track 1.